### DISCUSSION TITLE
ComfyUI Plugins List

### DISCUSSION THREAD

**Discussion by WASasquatch** [comfyanonymous/ComfyUI#470] [Show and tell]
I put up a repo to keep track of ComfyUI plugins. Feel free to PR your own or if I missed any.  Hopefully @comfyanonymous can link from the README so it can be easily found for people. 

https://github.com/WASasquatch/comfyui-plugins/blob/main/README.md

**Comment by alamonelf** [comment]
TY much love

**Comment by Lling00** [comment]
这个非常好，感谢分享~
